sys/cbor/cbor.c

```diff
@@ -236,12 +233,12 @@ void dump_memory(const unsigned char *data, size_t size)
         return;
     }
 
-    DEBUG("0x");
+    printf("0x");
```

Comment:
I'm not sure to be honest. If the respective functions are optimized out for non-dumping deserialization, then yes, if not no.